### ISSUE TITLE
Provide Label Property on Item Class

### ISSUE THREAD

**Issue by JeffFerguson** [JeffFerguson/gepsio#14]
The `Item` class should maintain a property of type `string` called `Label`, which will return the label from the Label linkbase matching the fact. If no label is available, then the property should evaluate to an empty string.

The sample available in [Issue 12](https://github.com/JeffFerguson/gepsio/issues/12) should be used as an implementation starting point for this work.

**Comment by ValeriyTitov** [comment]
I think string property is not best way to implement.

1. According article on page 33 in [Xbrl in plain english.pdf](http://www.batavia-xbrl.com/downloads/XBRLinPlainEnglishv1.1.pdf) each concept (fact) can have more than one label (depending on language, fact value, or other conditions).  I supose, label should be list of strings or even own class.

> Note that we give each label resource for the same concept the same value for the label attribute. This enables us to use a one-to-many link from a concept to all its labels. In this case three different types of label for the nr_employees_total concept.

2. Im unsure if im asking right question for finding labels for fact. 
Because "Flat list of facts with label" is not real life example of xbrl-report. Usually for me - its datatable, with Label in its header. 
However, as i understand, xbrl taxononmy can contain presentation level with description of best look.

**Comment by JeffFerguson** [comment]
Thank you for the feedback! I'll keep this in mind and read up a bit more before I finalize an implementation.
